Apps/W1/ShopifyConnector/app/app.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id":  "ec255f57-31d0-4ca2-b751-f2fa7c745abb",
+    "name":  "Connect to Shopify",
+    "publisher":  "Microsoft",
+    "brief":  "To be updated",
+    "description":  "To be updated",
+    "version":  "20.0.0.0",
```

Comment:
This PR is for version 20, while the GitHub branch is for version 19.3. This branch can hence to get merged as it is right now. But as this is a working branch, it should get merged. The PR is not ready.